backend/apps/owasp/services/certificate_providers.py

```
@@ -0,0 +1,74 @@
+"""Certificate provider classes for Contributor Recognition Program."""
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING
+
+from django.conf import settings
+
+from apps.owasp.models.crp.certificate import Certificate
+from apps.owasp.models.crp.recognition_enums import TierChoices
```

Ruff (0.15.15)

[warning] Move application import `apps.owasp.models.crp.recognition_enums.TierChoices` into a type-checking block

Move into type-checking block

(TC001)